src/platform/w3dengine/client/w3ddisplay.h

```diff
@@ -43,7 +48,12 @@ class W3DDisplay final : public Display
     virtual bool Is_Clipping_Enabled() override { return m_isClippedEnabled; }
```

Comment:
noticed // WB additional virtual here above, we need that for wb, i guess doAssetUsageDump or smth like that?
It dumps all asset usage to a txt

Reply:
Fixed, implemented both functions.